src/coffee/run.coffee

```diff
@@ -14,6 +14,7 @@ argv = require('optimist')
   .describe('clientSecret', 'your OAuth client secret for the SPHERE.IO API')
   .describe('accessToken', 'an OAuth access token for the SPHERE.IO API')
   .describe('sphereHost', 'SPHERE.IO API host to connect to')
+  .describe('sphereProtocol', 'SPHERE.IO API protocol to connecto to')
```

Comment:
there are already some defaults define with `.default(...)`:
```
./bin/stock-export
Usage: node ./bin/stock-export --projectKey [key] --clientId [id] --clientSecret [secret] --logDir [dir] --logLevel [level]

Options:
  --projectKey          your SPHERE.IO project-key                                       [required]
  --clientId            your OAuth client id for the SPHERE.IO API
  --clientSecret        your OAuth client secret for the SPHERE.IO API
  --accessToken         an OAuth access token for the SPHERE.IO API
  --sphereHost          SPHERE.IO API host to connect to
  --sphereProtocol      SPHERE.IO API protocol to connecto to
  --channelKey          when you want to filter stock entries for a special channel key
  --targetDir           the folder where exported files are saved                        [default: "/Users/yannsimon/dev/src/sphere-stock-export/exports"]
  --targetFile          the file where exported stocks are saved
  --useExportTmpDir     whether to use a system tmp folder to store exported files       [default: false]
  --logLevel            log level for file logging                                       [default: "info"]
  --logDir              directory to store logs                                          [default: "."]
  --logSilent           use console to print messages                                    [default: false]
  --excludeEmptyStocks                                                                   [default: false]
```

I actually do not want to define a default that way.
If I add a `(default: "https")` into the string, it will be very strange for the output.
And we do not define any default for the host as well.